src/main/target/KAKUTEF4/target.h

```diff
@@ -137,6 +137,7 @@
 
 #define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
 #define USE_ADC
+#define ADC1_DMA_STREAM 			DMA2_Stream0
 #define VBAT_ADC_PIN                PC3
 #define VBAT_ADC_CHANNEL            ADC_Channel_13
```

Comment:
You can remove `ADC_Channel_*` from `target.h`, as channels are inferred from corresponding pins.

Reply:
Yeah, i didn't pay attention to anything but DMA things, but right, channel lines are unnecessary.